core/frontend/src/main/scala/io/udash/properties/Properties.scala

```diff
@@ -2,6 +2,7 @@ package io.udash.properties
 
 trait Properties {
   implicit val allowDomElementTpe: ImmutableValue[org.scalajs.dom.Element] = null
+  implicit val allowJavaDateTpe: ImmutableValue[java.util.Date] = null
```

Comment:
https://github.com/scala-js/scala-js/blob/767a6daff9831f31c28508d638c7bf80190f4232/javalib/src/main/scala/java/util/Date.scala

with these setters it's not all that immutable